netci.groovy

```diff
@@ -1512,6 +1543,37 @@ combinedScenarios.each { scenario ->
                                         buildCommands += "build.cmd ${lowerConfiguration} ${arch} skiptests"
                                         buildCommands += "set __TestIntermediateDir=int&&build-test.cmd ${lowerConfiguration} ${arch}"
                                     }
+                                    else if (scenario == 'formatting') {
+                                        // Build with NMake Makefiles for all configurations
+                                        buildCommands += "set __TestIntermediateDir=int&&build.cmd Debug ${arch} usenmakemakefiles"
```

Comment:
This logic is significantly harder to test embedded into the groovy script like this. Can you hoist this out into a cross platform script and just call that?